src/units/abilities.cpp

```diff
@@ -1556,7 +1556,7 @@ bool attack_type::special_active_impl(const_attack_ptr self_attack, const_attack
 	temporary_facing self_facing(self, self_loc.get_relative_dir(other_loc));
 	temporary_facing other_facing(other, other_loc.get_relative_dir(self_loc));
 
-	// Filter poison, plague, drain, first strike
+	// Filter poison, plague, drain, slow, petrifies
```

Comment:
The parts looking at slow and petrified seem unchanged, so this comment shouldn't have removed them, I think?

Reply:
slow and petrified added recently but no specified in comment, i use this PR to fix this.

Reply:
Ah, read the diff backwards. It should still mention firststrike though then?

Reply:
This section is now the status-aliment abilities which are countered by unpoisonable, unslowable, etc, and `drain` which is handled in the same way as the status-aliment abilities.

The reason for moving firststrike later in the file was to get access to the `att_weapon` special, but it's also the case that the logic for filtering it is different to the logic in this block.